go.mod

```diff
@@ -3,6 +3,9 @@ module github.com/siderolabs/omni
 go 1.23.3
 
 replace (
+	// todo: remove when merged
+	github.com/siderolabs/omni-infra-provider-bare-metal v0.0.0-20241014092637-c3e892b87244 => github.com/utkuozdemir/sidero-omni-infra-provider-bare-metal v0.0.0-20241021213554-cc93e594a0d2
```

Comment:
I think if we need an API, it should be common for all providers, and should either live in Omni or in a separate repo.

my 2 cents

Reply:
Yes you are right, I'll anyway keep this as draft until we make a decision.